tests/cross_functional/system_test/test_nfs_export_in_out_cluster.py

```
@@ -0,0 +1,2001 @@
+                if deployment_obj.is_exist(resource_name=pod_name):
+                    deployment_obj.delete(resource_name=pod_name)
+                    deployment_obj.wait_for_delete(resource_name=pod_name, timeout=180)
+                    log.info(f"Deployment {pod_name} deleted successfully")
+            except Exception as e:
+                log.warning(f"Failed to delete deployment: {e}")
+
+            # 3. Wait for pod termination
+            try:
+                log.info("Waiting for pod to be terminated...")
+                pod_obj.ocp.wait_for_delete(pod_obj.name, timeout=180)
+                log.info(f"Pod {pod_obj.name} terminated successfully")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)